sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -69,23 +66,17 @@ case class CreateViewCommand(
 
   override def output: Seq[Attribute] = Seq.empty[Attribute]
 
-  if (!isTemporary) {
```

Comment:
`originalText` is empty only when the view is created through Dataset APIs, as shown in the [code comment ](https://github.com/gatorsmile/spark/blob/5606344584c1c45cf49d2a95158c55193e777a9e/sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala#L40-L41)

This checking looks useless. If you think we should keep it, I can add it back. Thanks!